app/src/test/java/comp3350/mealbuddy/tests/objects/ExerciseTest.java

```diff
@@ -0,0 +1,20 @@
+package comp3350.mealbuddy.tests.objects;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import comp3350.mealbuddy.objects.Exercise;
+
+
+public class ExerciseTest{
+    public static final double DURATION_TESTVALUE = 10.00;
+    public static final double ASSERT_DELTAPRECISION = 0.000000001;
+
+    public void test_exerciseConstructor() {
```

Comment:
Don't use JUnit 3 use 4. We need to be consistent

Reply:
Dont start every test name with test.  Thats not the convention we are following. we are doing method_situation_result